bn254-fr/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "p3-bn254-fr"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+p3-field = { path = "../field" }
+p3-poseidon2 = { path = "../poseidon2" }
+p3-symmetric = { path = "../symmetric" }
+
+ff = { version = "0.13", features = ["derive", "derive_bits"] }
+lazy_static = "1.4"
```

Comment:
I think lazy_static isn't that popular now that parts of once_cell are in std; in particular I think we can use `core::cell::OnceCell`?

Reply:
@dlubarov - To get an atomic version of `OnceCell`, I did have to use `OnceLock`, which does require `std`.  Here are the [commits](https://github.com/Plonky3/Plonky3/pull/279/files/26dbde0b7e09d87f12010b055251f937719a6f58..3566997618dac49cfe80e76c60962507373509f9) made.

I wanted to double check with you before merging about the usage of `std`, as I believe this would make the `bn-254-fr` package the first one in `plonky3` to use it.

Reply:
Oh I see, I guess it's fine since any wasm users probably wouldn't use this crate, or `lazy_static` seems fine also